_redirects

```diff
@@ -2,3 +2,31 @@ http://rmarkdown.rstudio.com/*   https://rmarkdown.rstudio.com/:splat  301!
 https://rmarkdown.netlify.com/*  https://rmarkdown.rstudio.com/:splat  301!
 http://rmarkdown.netlify.com/*   https://rmarkdown.rstudio.com/:splat  301!
 /flexdashboard    /flexdashboard/   301!
+
+/r_notebooks.html  https://bookdown.org/yihui/rmarkdown/notebook.html  301!
```

Comment:
Section header levels changed in the book (n -> n + 1).

Added figure/table captions, numbers, and cross-references.